### ISSUE TITLE
Give a pat on your back, you well deserve it.

### ISSUE THREAD

**Issue by manasan3010** [sabrogden/Ditto#480]
I am a longtime ditto user and I can't even explain how much time and pain your software has saved me from. I've been using it for over three years now, and it hasn't failed me on any occasion NOT EVEN ONE. I couldn't find even a single bug even after heavy daily usage for three years. You have sculpted this software inch by inch to achieve near perfection. I can well understand the amount of time and thoughts you've put into every single button and option in the software. You might not receive an award from some shitty agency but you have won my heart and your software will remain in my memories forever. 

You should be proud of yourself, you've gifted the mankind with a great tool.
My only request to you is, Please Don't Become Evil ;)

**Comment by sjvudp** [comment]
Please understand what an issue tracker is: It's not a discussion forum. You can contact the author using other means if you need. I also guess the author would not reject a donation from you ;-)

**Comment by Chealer** [comment]
Indeed, this is an ITS; please discard this ticket and use [the forums](https://sourceforge.net/p/ditto-cp/discussion/) instead (and indeed, there are more concrete ways to give back).
